Support latest React Native versions

**Describe the bug**

This has been a great and convenient component.

Unfortunately, when using it with the latest react-native versions (greater than .64) we get peer dependency errors.
We have to `npm --force` to resolve them. The latest react-native version is 0.68.2.

We're using react-native 0.67.2. This package is configured to only support up to .64 but appears to work fine if forced.

```
Could not resolve dependency:
npm ERR! peer react-native@">=0.60 <=0.64" from @api.video/react-native-player@1.0.0
```

**To Reproduce**
Steps to reproduce the behavior:
1. npm i  react-native@0.67.2
2. npm i @api.video/react-native-player@1.0.0

**Expected behavior**
No peer dependency errors please :)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,9 @@
 # Changelog
 All changes to this project will be documented in this file.
 
+## [1.0.1] - 2022-06-13
+- Remove react-native peer dependency max version constraint
+
 ## [1.0.0] - 2021-12-13
 - Allow autoplay on not muted videos
 

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@api.video/react-native-player",
   "author": "api.video",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "React Native api.video player",
   "repository": {
     "type": "git",
@@ -30,7 +30,7 @@
     "postinstall": "node ./scripts/add-native-deps.js"
   },
   "peerDependencies": {
-    "react-native": ">=0.60 <=0.64"
+    "react-native": ">=0.60"
   },
   "husky": {
     "hooks": {

